frappe/utils/print_format.py

```diff
@@ -63,23 +92,35 @@ def download_multi_pdf(doctype, name, format=None, no_letterhead=False, letterhe
 
 	if not isinstance(doctype, dict):
 		result = json.loads(name)
+		total_docs = len(result)
 
 		# Concatenating pdf files
-		for ss in result:
-			pdf_writer = frappe.get_print(
-				doctype,
-				ss,
-				format,
-				as_pdf=True,
-				output=pdf_writer,
-				no_letterhead=no_letterhead,
-				letterhead=letterhead,
-				pdf_options=options,
+		for idx, ss in enumerate(result):
+			try:
+				pdf_writer = frappe.get_print(
+					doctype,
+					ss,
+					format,
+					as_pdf=True,
+					output=pdf_writer,
+					no_letterhead=no_letterhead,
+					letterhead=letterhead,
+					pdf_options=options,
+				)
+			except Exception:
+				frappe.publish_realtime(task_id=task_id, message={"message": "Failed"})
+
+			# Publish progress
+			frappe.publish_progress(
+				percent=(idx + 1) / total_docs * 100,
+				title=_("Please keep this tab open until the operation is complete."),
```

Comment:
This should be in description, not in title.

Reply:
Description felt a bit too long then, need to figure out what all formatting can be used in the description.
Couldn't use `<br>` or `\n` for newlines

Reply:
It's definitely not title worthy. Long description is fine ig.